packages/grafana-ui/src/components/Drawer/Drawer.story.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Drawer } from './Drawer';
+import { UseState } from '../../utils/storybook/UseState';
+import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+import mdx from './Drawer.mdx';
+
+export default {
+  title: 'UI/Drawer',
+  component: Drawer,
+  decorators: [withCenteredStory],
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const global = () => {
+  return (
+    <UseState initialState={{ isOpen: false }}>
+      {(state, updateValue) => {
+        return (
+          <>
+            <div
+              style={{ border: '1px solid gray', borderRadius: '4px', padding: '10px', cursor: 'pointer' }}
+              onClick={() => updateValue({ isOpen: !state.isOpen })}
+            >
+              Open drawer
+            </div>
+            {state.isOpen && (
+              <Drawer
+                title="storybook"
```

Comment:
Can we add a knob for that? Can React element be passed as a title?

Reply:
I don' think we should allow a react element to be passed to title, maybe further down the line if there's some use case for it.

Reply:
agree